client/src/components/main/splash.scss

```diff
@@ -29,17 +29,6 @@ $background-image: 'https://roastia.s3.us-east-2.amazonaws.com/roastia-home.jpeg
   };
 }
 
-#filter-icon {
```

Comment:
Wanted to clarify the reasoning behind removing this. I know it might look unnecessary, but I spent a lot of time making sure the icons stayed above the fold on different screen sizes, so wanted to see the thought process here.

I definitely could be missing a better way of doing it, but this worked on both small and large screens.

Reply:
Yeah I tried having all of those icons be the same size and the filter looked aweful, the filter is being defined with a custom stylesheet here now
https://github.com/hegelocampus/Roastia/blob/152b2f4a1ae2feea3a62725850d288bd2aa4bb6b/client/src/components/util/icons.js#L25-L34

Reply:
haha yep same here. Awesome, thanks for explaining.